index.html

```diff
@@ -995,7 +998,7 @@ <h2>
         Gap Analysis
       </h2>
       <p>
-        Please look at <a href="comparison.html">the comparison table of APIs in MiniApps, W3C specs, and PWAs</a>.
+        Learn more about the <a href="comparison.html">differences between MiniApp and PWA</a>, considering the status of the W3C specifications. Also, find a summary of the <a href="components-comparison.html">differences between MiniApp components</a> in the different MiniApp versions compared to Web Components. 
```

Comment:
"MiniApp user agents / implementations" might be more appropriate than "MiniApp versions" here.

Reply:
Good point, thanks. Changed at 4994bb011c7f832a6346f6fdaf6eb122c63fbffe